modules/reactor/src/meshgenerators/PatternedCartesianPeripheralModifier.C

```diff
@@ -32,7 +32,7 @@ PatternedCartesianPeripheralModifier::PatternedCartesianPeripheralModifier(
 std::unique_ptr<MeshBase>
 PatternedCartesianPeripheralModifier::generate()
 {
-  if (hasMeshProperty("square_center_trimmability", _input_name))
+  if (hasMeshProperty<Real>("square_center_trimmability", _input_name))
```

Comment:
bool ?
why isnt it erroring?

Reply:
This is good now, should have been bool.